crates/holochain/src/conductor/conductor/tests/state_dump.rs

```diff
@@ -0,0 +1,69 @@
+use std::time::Duration;
+
+use holo_hash::ActionHash;
+use holochain_conductor_api::FullStateDump;
+use holochain_state::source_chain;
+use holochain_wasm_test_utils::TestWasm;
+
+use crate::{
+    conductor::{conductor::state_dump_helpers::peer_store_dump, full_integration_dump},
+    sweettest::{SweetConductor, SweetDnaFile, SweetZome},
+};
+
+#[tokio::test(flavor = "multi_thread")]
+async fn dump_full_state() {
+    let mut conductor = SweetConductor::from_standard_config().await;
+    let dna_file = SweetDnaFile::unique_from_test_wasms(vec![TestWasm::Crd])
+        .await
+        .0;
+    let app = conductor.setup_app("", &[dna_file]).await.unwrap();
+    let cell_id = app.cells()[0].cell_id();
+    let _: ActionHash = conductor
+        .call(
+            &SweetZome::new(cell_id.clone(), TestWasm::Crd.coordinator_zome_name()),
+            "create",
+            (),
+        )
+        .await;
+    // Await integration.
+    tokio::time::timeout(Duration::from_secs(1), async {
```

Comment:
Should this be extracted into a sweetest utility?

Reply:
Yes, it should be, like it is in kitsune2.

Reply:
Done https://github.com/holochain/holochain/pull/4817/commits/107c5fc49757e54edf31b01d1632c3ce9ffc3a3a

Reply:
Oh, that's good. I was referring to the inner part as well -- a utility that waits until the number of un-integrated authored ops is 0

Reply:
Ah! Mmh, that's like awaiting consistency for just one conductor. I have no clue if that's used anywhere else.